packages/frontend/src/components/AppBar.tsx

```diff
@@ -0,0 +1,26 @@
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+
+function AppBarComponent() {
+    return (
+        <header>
+            <nav>
+                <AppBar position="static">
+                    <Toolbar>
+                        <Typography
+                            variant="h6"
+                            component="a"
+                            href="/"
+                            sx={{ flexGrow: 1, textDecoration: 'none', color: 'inherit' }}
+                        >
+                            Shoodies
+                        </Typography>
+                    </Toolbar>
+                </AppBar>
+            </nav>
+        </header>

```

Comment:
```suggestion
        <AppBar component="header" role="banner" position="static">
            <Toolbar>
                <Typography
                    variant="h6"
                    component="a"
                    href="/"
                    sx={{ flexGrow: 1, textDecoration: 'none', color: 'inherit' }}
                >
                    Shoodies
                </Typography>
            </Toolbar>
        </AppBar>
```
Here nav is not necessary because it will be used, mainly, with a role "navigation".
When we will add links in AppBar (Eg: Home, Contact, About etc..) it will be useful to add it inside a nav html element. For now we don't need it.

Reply:
resolved